api/samples/memval_simple.c

```diff
@@ -403,7 +421,7 @@ event_exit(void)
 DR_EXPORT void
 dr_client_main(client_id_t id, int argc, const char *argv[])
 {
-    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, false};
+    drreg_options_t ops = {sizeof(ops), 3 /*max slots needed*/, true};
```

Comment:
If it were dead why would we need the value?  You'd only need it if being dead is somehow wrong: e.g., if the app goes and reads the seemingly-dead value in a fault handler: an extreme corner case we have never seen.

It should be impossible to be simultaneously dead and to satisfy your opnd_uses_reg check, right?